src/Type/Php/RegexArrayShapeMatcher.php

```diff
@@ -620,6 +632,7 @@ private function walkGroupAst(TreeNode $ast, TrinaryLogic &$isNonEmpty, TrinaryL
 					$literalValue = $this->getLiteralValue($child);
 
 					if ($literalValue === null) {
+						$allNumeric = false;
```

Comment:
at this point, I think we can just drop the 2 `$allNumeric = false;` occurences

Reply:
Ok done in 292e21147 let's see if the build agrees :D